docs/source/host/maintain-domain/view-logs.rst

```diff
@@ -1,15 +1,11 @@
-.. warning::
-    This document is outdated.
-    FIXME: mentions High Fidelity, paths outdated, missing path for Linux servers
-
 #######################
 View Domain Server Logs
 #######################
 
 Your domain server logs list the different events relevant to your High Fidelity domain. These can help you troubleshoot any issues with your domain and understand how the different parts of your domain communicate with each other.
 
-Server logs are stored directly on the server in the following directories:
+Server logs are stored directly on the server and can be accessed as follows:
 
-* **Windows**: %AppData%/Local/High Fidelity/Server Console/logs
-* **Mac**: ~/Library/Application Support/High Fidelity/Server Console/Logs
+* **Windows**: Logs can be found in the directory "%AppData%/Local/Vircadia/Server Console/logs"
```

Comment:
```suggestion
* **Windows**: Logs can be found in the directory "%AppData%\Local\Vircadia\Server Console\logs"
```
Windows uses backslashes instead of slashes.